doc/classes/Control.xml

```diff
@@ -223,6 +232,18 @@
 				[b]Note:[/b] This does not affect the methods in [Input], only the way events are propagated.
 			</description>
 		</method>
+		<method name="accessibility_drag">
+			<return type="void" />
+			<description>
+				Starts drag-and-drop operation.
+			</description>
+		</method>
+		<method name="accessibility_drop">
+			<return type="void" />
+			<description>
+				Ends drag-and-drop operation.
+			</description>
+		</method>
```

Comment:
It's not obvious from the description how this relates to accessibility.

Reply:
It's not necessarily accessibility specific, just for implementing drag-and-drop without using a mouse.